packages/@azure/servicebus/data-plane/test/sessionsTests.spec.ts

```diff
@@ -0,0 +1,494 @@
+  });
+
+  it("Partitioned Topics and Subscription with Sessions - Batch Receiver: complete() removes message", async function(): Promise<
+    void
+  > {
+    await testComplete_batching(partitionedTopicSessionClient, partitionedSubscriptionSessionClient);
+  });
+
+  it("Unpartitioned Queues with Sessions - Batch Receiver: complete() removes message", async function(): Promise<
+    void
+  > {
+    await testComplete_batching(unpartitionedQueueSessionClient, unpartitionedQueueSessionClient);
+  });
+
+  it("Unpartitioned Topics and Subscription with Sessions - Batch Receiver: complete() removes message", async function(): Promise<
+    void
+  > {
+    await testComplete_batching(unpartitionedTopicSessionClient, unpartitionedSubscriptionSessionClient);
+  });
+
+  async function testComplete_streaming(
+    senderClient: QueueClient | TopicClient,
+    sessionClient: QueueClient | SubscriptionClient
+  ): Promise<void> {
+
+    await senderClient.send(testMessagesWithSessions[0]);
+
+    const receiverClient = await sessionClient.acceptSession();
+    const receivedMsgs: ServiceBusMessage[] = [];
+    receiverClient.receive((messageSession: MessageSession, msg: ServiceBusMessage) => {
+      receivedMsgs.push(msg);
+      should.equal(msg.body, testMessagesWithSessions[0].body);
+      should.equal(msg.messageId, testMessagesWithSessions[0].messageId);
+      return Promise.resolve();
+    }, unExpectedErrorHandler);
+
+    for (let i = 0; i < 5; i++) {
+      await delay(1000);
+      if (receivedMsgs.length === testMessagesWithSessions.length) {
+        break;
+      }
+    }
+    should.equal(unexpectedError, undefined, unexpectedError && unexpectedError.message);
+
+    await testPeekMsgsLength(receiverClient, 0);
+  }
+
+
+  it("Partitioned Queues with Sessions - Streaming Receiver: complete() removes message", async function(): Promise<
+    void
+  > {
+    await testComplete_streaming(partitionedQueueSessionClient, partitionedQueueSessionClient);
+  });
+
+  it("Partitioned Topics and Subscription with Sessions - Streaming Receiver: complete() removes message", async function(): Promise<
+    void
+  > {
+    await testComplete_streaming(partitionedTopicSessionClient, partitionedSubscriptionSessionClient);
+  });
+
+  it("Unpartitioned Queues with Sessions - Streaming Receiver: complete() removes message", async function(): Promise<
+    void
+  > {
+    await testComplete_streaming(unpartitionedQueueSessionClient, unpartitionedQueueSessionClient);
+  });
+
+  it("Unpartitioned Topics and Subscription with Sessions - Streaming Receiver: complete() removes message", async function(): Promise<
+    void
+  > {
+    await testComplete_streaming(unpartitionedTopicSessionClient, unpartitionedSubscriptionSessionClient);
+  });
+});
+
+describe("Accept a session by passing non-existing sessionId - Queue/Subscription has messages belonging to same sessionId", function(): void {
```

Comment:
We can remove the previous describe block.
Because, for clean up here in the second describe block, you are using the exact code as in the above section

Reply:
Also rename this to `Accept a session by passing non-existing sessionId receives no messages`

Reply:
The only change is - For cleaning(in the second describe), I'm passing the sessionId to accept the session!
@ramya-rao-a 
So, I'll update the cleaning part to "receiving messages by not passing sessionId" and then remove the first describe block ?

Reply:
Yes

Reply:
Done!